eclide/WorkunitView.cpp

```diff
@@ -293,11 +293,19 @@ void CWorkunitTreeView::Update(HTREEITEM hItem, Dali::IWorkunitVectorCom* wus)
     if ( !wus->empty() || hItem == TVI_ROOT )
     {
         //delete all the workunits that are under this parent node
+        CString strEmpty;
+        strEmpty.LoadString(IDS_TREE_EMPTY);
+
         for(WTL::CTreeItem curr = node.GetChild(); !curr.IsNull(); )
         {
             CTreeNode *node = reinterpret_cast<CTreeNode *>(curr.GetData());
             CComQIPtr<CWorkunitNode> wuNode = node;
```

Comment:
I think if you just changed this one line to:
```
CComQIPtr<CTreeNode> treeNode = node;
```
It will catch both ```CWorkunitNode``` and ```CLoadingNode``` which are both derived from it.

Reply:
@GordonSmith I thought of the same thing late last night. Will change.